engine_gui/include/gui/XWWindow.h

```diff
@@ -1,5 +1,5 @@
-#ifndef RCBE_ENGINE_WINDOW_H
-#define RCBE_ENGINE_WINDOW_H
+#ifndef RCBE_ENGINE_XWWINDOW_H
```

Comment:
Revert this as this is a side-effect of CLion's auto-replace

Reply:
I will remove this whole package in a followup request, since this is obsolete.